src/models/Event.js

```diff
@@ -39,27 +40,38 @@ const eventSchema = new Schema({
                     validator: (value) => validTime.test(value),
                     message: 'Invalid time format! Example: hh:mm (24h)'
                 }
-                
             }
-        }, { // Additional date and time (if needed)
-            date: { type: Date },
-            startTime: { type: String },
-            endTime: { type: String }
         }]
     },
-    imageUrl: { type: String, required: true },
+    imageUrl: {
+        type: String,
+        required: true,
+        validate: {
+            validator: (value) => validString.test(value),
+            message: "Invalid URL, must start with HTTP:///HTTPS://",
+        },
+    },
     contacts: {
         // TODO: Check later for unique COORDS..!
-        coordinates: { lat: { type: String }, long: { type: String } },
+        coordinates: { lat: { type: String, required: true }, long: { type: String, required: true } },
```

Comment:
Nobody’s going to know the event’s place coordinates. We must convert them, they are not going to come as an input

Reply:
Sorry i dod not understand you can you rephrase yourself?